src/com/theironyard/novauc/Main.java

```diff
@@ -1,8 +1,126 @@
 package com.theironyard.novauc;
 
+import java.util.*;
+import java.text.NumberFormat;
+import java.math.*;
+import java.util.ArrayList;
+
 public class Main {
 
-    public static void main(String[] args) {
-	// write your code here
-    }
-}
+    public static void main(String[] args) throws Exception {
+
+        // write your code here
+
+//
+         ArrayList options = new ArrayList();
+
+        System.out.println("Good Morning!");
+
+        System.out.println("What is your name?");
+
+        Scanner scanner = new Scanner(System.in);
+
+        String name = scanner.nextLine();
+
+        System.out.println(" Welcome," + name);
+
+        HashMap<String, Double> account = new HashMap<>();
+        account.put("Mary", new Double(300.59) );
+        account.put("Johnny", new Double(1500.36));
+        account.put("Walter", new Double(2252.39));
+        account.put("Tony", new Double(398));
+
+        System.out.println("Check if Name is in Hashmap: " + account.containsKey(name));
+        if(account.containsKey(name)){
+
+        } else
+        System.out.println("Would like to create an account? ");
+        String create = scanner.nextLine();
+
+        while (true){
+            System.out.println("1.Check Balance:");
+            System.out.println("2. Withdraw Funds: ");
+            System.out.println("3.Cancel: ");
+            System.out.println("4. Remove account:");
+            String text = scanner.nextLine();
+
+            if(text.equals("1")){
+                System.out.println("Your balance is:" + account.get(name) );
+//                String option = scanner.nextLine();
+
+            } else if(text.equals("2")) {
+                System.out.println("Please input amount to withdraw:");
+                String answer = scanner.nextLine();
+                Double to = Double.valueOf(answer);
+                Double me = account.get(name) - to;
+                System.out.println("Please take your cash ");
+                account.put(name, me);
+                System.out.println("Your new balance" + account.get(name));
+
+
+            }
+                else if(text.equals("4")) {
+                System.out.println("Your account has been removed" + account.remove(name));
+//                String been = scanner.nextLine();
```

Comment:
Line 63 prints the value of key you are removing.  So account.remove(name) should happen on a separate line not within the print statement.